contracts/HATVaults.sol

```diff
@@ -44,7 +43,9 @@ import "./Governable.sol";
 // HVE32: Wrong amount received
 // HVE33: Reward level can not be more than 10000
 // HVE34: LP token is zero
-// HVE35: Token approve reset failed
+// HVE35: Only fee setter
+// HVE36: Fee must be less than 2%
+// HVE37: Token approve reset failed
 contract  HATVaults is Governable, HATMaster {
```

Comment:
Ah this still works even if you do not import it? Anyay, I'd remove the `Governanable` dependency here for clarity